portals/api-portal/src/defaultContent/styles/components.css

```
@@ -12,6 +12,18 @@
   white-space: nowrap;
 }
 
+.dp-badge-icon { width: 0.875em; height: 0.875em; flex: none; }
+
+.mcp-glyph {
+  display: inline-block;
+  flex: none;
+  width: 1em;
+  height: 1em;
+  background-color: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)